### PR TITLE
Add test case to external IDP generated JWT

### PR DESCRIPTION
**Pull request by tharindu1st**

### CODE REVIEW

**Comment by tharindu1st** [comment]
depends on https://github.com/wso2/carbon-apimgt/pull/8055

File: modules/integration/tests-integration/tests-backend/pom.xml [hunk 1]
```diff
@@ -702,5 +731,11 @@ under the License.
             <artifactId>json-simple</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.wso2.orbit.com.nimbusds</groupId>
+            <artifactId>nimbus-jose-jwt</artifactId>
+            <version>7.3.0.wso2v1</version>
```
Comment (bhathiya) [inline modules/integration/tests-integration/tests-backend/pom.xml]: please move the versioned dependency to the root pom.

File: pom.xml [hunk 2]
```diff
@@ -1182,7 +1182,7 @@
         <carbon.analytics.common.version>5.2.16</carbon.analytics.common.version>
 
         <!-- APIM Component Version -->
-        <carbon.apimgt.version>6.5.442</carbon.apimgt.version>
+        <carbon.apimgt.version>6.5.445-SNAPSHOT</carbon.apimgt.version>
```
Comment (bhathiya) [inline pom.xml]: snapshot?

File: modules/integration/tests-integration/tests-backend/pom.xml [hunk 3]
```diff
@@ -651,16 +681,16 @@ under the License.
...
+<!--        <dependency>-->
+<!--            <groupId>org.slf4j</groupId>-->
+<!--            <artifactId>slf4j-jdk14</artifactId>-->
+<!--            <version>1.7.28</version>-->
+<!--        </dependency>-->
+<!--        <dependency>-->
+<!--            <groupId>org.slf4j</groupId>-->
+<!--            <artifactId>log4j-over-slf4j</artifactId>-->
+<!--            <version>1.7.28</version>-->
+<!--        </dependency>-->
```
Comment (bhathiya) [inline modules/integration/tests-integration/tests-backend/pom.xml]: shall we remove these?

**Pull request merged**